syft/workers/base.py

```diff
@@ -166,6 +166,8 @@ def __init__(
         # storage object for crypto primitives
         self.crypto_store = PrimitiveStorage(owner=self)
 
+        self.syft = sy
```

Comment:
Q: Where is this needed? Can't we access it with ```worker.hook.syft```?

Reply:
Yeah it's a trick, because when you will register with @ allow_command a function "syft.foo.bar", a worker executing the command will try to execute it as a method so "worker.syft.foo.bar", just like we to for torch, when you do "torch.add" it tries "worker.torch.add" that's why I added it...